sympy/stats/tests/test_stochastic_process.py

```diff
@@ -128,7 +153,7 @@ def test_sample_stochastic_process():
     if numpy:
         numpy.random.seed(0) # scipy uses numpy to sample so to set its seed
     T = Matrix([[0.5, 0.2, 0.3],[0.2, 0.5, 0.3],[0.2, 0.3, 0.5]])
-    Y = DiscreteMarkovChain("Y", [0, 1, 2], T)
+    Y = DiscreteMarkovChain("Y", ['A', 2, 3], T)
```

Comment:
I'd suggest to keep the old test and add new ones.

Reply:
Done with that one and I think one other. I at least want to change some since creating new tests will increase the number of lines by a drastic amount for very little gain. 

If a test passes with the new states, then it would certainly pass using the old states. But the converse does not hold.